api/cluster.go

```diff
@@ -420,7 +423,9 @@ func FetchClusters(c *gin.Context) {
 
 	var clusters []model.ClusterModel //TODO change this to CommonClusterStatus
 	db := model.GetDB()
-	err := db.Find(&clusters).Error
+	organization := auth.GetCurrentOrganization(c.Request)
+	organization.Name = ""
```

Comment:
Is this needed here? By default string will be initialized as ""

Reply:
This is not a simple object, it's parsed from the request. For further query the name is ignored only the Id used